### ISSUE TITLE
The loss curve fluctuates greatly

### ISSUE THREAD

**Issue by COST-97** [hpcaitech/Open-Sora#413]
Hello:
I found that the loss curve fluctuates greatly.
 Based on the model fine-tuning of opensora1.1, I tried different batch sizes of 32 and 64, learning rates of 2e-5, 1e-6, optimizer beta values (0.95, 0.9995), etc., but this problem has always existed.
 Do you have any suggestions on how to mitigate this problem?

**Comment by SkylerZheng** [comment]
Wondering what learning rates were used for each stage video generation model training?
